### PR TITLE
3-16 work

### PR DESCRIPTION
**Pull request by KangarooKoala**

### CODE REVIEW

File: src/main/java/frc/team2412/robot/util/autonomous/AutonomousChooser.java [hunk 1]
```diff
@@ -114,9 +113,10 @@ private static SequentialCommandGroup getLineAutoCommand(Subsystems subsystems,
                         Subsystems.SubsystemConstants.SHOOTER_VISION_ENABLED &&
                         Subsystems.SubsystemConstants.DRIVE_ENABLED &&
                         Subsystems.SubsystemConstants.INTAKE_ENABLED),
-        TWO_SCUFFED((subsystems, trajectories) -> new TwoBallScuffedAutoCommand(subsystems.indexSubsystem,
-                subsystems.shooterSubsystem, subsystems.shooterVisionSubsystem, subsystems.drivebaseSubsystem,
-                subsystems.intakeSubsystem), "TWO SCUFFED", true),
+        // TWO_SCUFFED((subsystems, trajectories) -> new
+        // TwoBallScuffedAutoCommand(subsystems.indexSubsystem,
+        // subsystems.shooterSubsystem, subsystems.shooterVisionSubsystem, subsystems.drivebaseSubsystem,
+        // subsystems.intakeSubsystem), "TWO SCUFFED", true),
```
Comment (TAKBS2412) [inline src/main/java/frc/team2412/robot/util/autonomous/AutonomousChooser.java:119]: I think it's safe to just delete this, it's already being deleted in other PRs.

**Review by TAKBS2412 — CHANGES_REQUESTED** [review]

**Review by TAKBS2412 — DISMISSED** [review]

File: src/main/deploy/shooterData.csv [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-60,25,2000
-81,32,2000
-110,38,2100
-142,42,2200
-183,50,2600
-220,53,3000
-300,58,3700
-332,60,4200
\ No newline at end of file
+# Distance (inches), Angle (degrees), Velocity (RPM)
```
Comment (bobtseattle) [inline src/main/deploy/shooterData.csv:1]: We need to make sure these values don't overwrite data points gathered 3/17.

File: src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java [hunk 3]
```diff
@@ -22,8 +22,8 @@
     // Constants
     public static class IntakeConstants {
 
-        public static final double INTAKE_IN_SPEED = 0.7;
-        public static final double INTAKE_OUT_SPEED = -0.7; // will adjust later after testing?
+        public static final double INTAKE_IN_SPEED = -0.3;
```
Comment (bobtseattle) [inline src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java:25]: We don't want these to conflict with tuned settings from 3/17.

File: src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java [hunk 4]
```diff
@@ -16,6 +16,7 @@
         public static final double HEIGHT_TO_RIM = RIM_HEIGHT - LIMELIGHT_HEIGHT_OFFSET;
         public static final double HUB_RADIUS = 24;
         public static final double LIMELIGHT_ANGLE_OFFSET = Math.toDegrees(Math.atan2(HEIGHT_TO_RIM, 360 - HUB_RADIUS)); // 10.95
+        // new LL crosshair angle is -0.7375
```
Comment (bobtseattle) [inline src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java:19]: Is this still the case after inverting the Limelight?

**Review by bobtseattle — COMMENTED** [review]
I noted some sections that might conflict with subsequent testing and tuning.

File: src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java (hunk 4 above)
Comment (TAKBS2412) [reply src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java:19]: It's not, this is the old limelight value

File: src/main/deploy/shooterData.csv [hunk 5]
```diff
@@ -1,18 +1,14 @@
-Distance (inches),Angle (degrees),Velocity (RPM)
```
Comment (TAKBS2412) [inline src/main/deploy/shooterData.csv:1]: We should be careful about merging this file because it contains older data points than #140.

**Review by TAKBS2412 — COMMENTED** [review]
The shooter data distance points are outdated but there are a lot of useful features here worth merging (csv comments, null checks in TargetLocalizer).

**Review by bobtseattle — APPROVED** [review]

**Pull request merged**